source/slang/glsl.meta.slang

```diff
@@ -2836,58 +2836,20 @@ void typeRequireChecks_shader_subgroup_GLSL() {
     __target_switch
     {
     case glsl:
-        if (__type_equals<T, half>()) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_float16");
-        else if (__type_equals<T, float16_t>()) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_float16");
-        else if (__type_equals<T, uint8_t>()) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_int8");
-        else if (__type_equals<T, uint16_t>()) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_int16");
-        else if (__type_equals<T, uint64_t>()) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_int64");
-        else if (__type_equals<T, int8_t>()) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_int8");
-        else if (__type_equals<T, int16_t>()) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_int16");
-        else if (__type_equals<T, int64_t>()) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_int64");
-        __intrinsic_asm "";
-    }
-}
-
-__generic<T : __BuiltinType>
-[require(spirv)]
-void typeRequireChecks_shader_subgroup_SPIRV() {
-    __target_switch
-    {
-    case spirv:
-
-        if (__type_equals<T, half>()) {
-
-            spirv_asm {OpCapability Float16};
-        }
-        else if (__type_equals<T, float16_t>()) {
-
-            spirv_asm {OpCapability Float16};
-        }
-        else if (__type_equals<T, uint8_t>()) {
+        if (__type_equals<T, half>()
+            || __type_equals<T, float16_t>()
+            ) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_float16");
+        else if (__type_equals<T, uint8_t>()
+            || __type_equals<T, int8_t>()
+            ) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_int8");
+        else if (__type_equals<T, uint16_t>()
+            || __type_equals<T, int16_t>()
+            ) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_int16");
+        else if (__type_equals<T, uint64_t>()
+            || __type_equals<T, int64_t>()
+            ) __requireGLSLExtension("GL_EXT_shader_subgroup_extended_types_int64");
 
-            spirv_asm {OpCapability Int8};
-        }
-        else if (__type_equals<T, uint16_t>()) {
-
-            spirv_asm {OpCapability Int16};
-        }
-        else if (__type_equals<T, uint64_t>()) {
-
-            spirv_asm {OpCapability Int64};
-        }
-        else if (__type_equals<T, int8_t>()) {
-
-            spirv_asm {OpCapability Int8};
-        }
-        else if (__type_equals<T, int16_t>()) {
-
-            spirv_asm {OpCapability Int16};
-        }
-        else if (__type_equals<T, int64_t>()) {
-
-            spirv_asm {OpCapability Int64};
-        }
-        else spirv_asm {};
+        __intrinsic_asm "";
```

Comment:
This can just be `return`.

Reply:
without outputting at a minimum intrinsic_asm (just `return` does not work) there is a problem with `internal error 99999: unimplemented feature in Slang compiler`